src/minter-suite-mapping.ts

```diff
@@ -95,6 +95,14 @@ import {
   UnregisteredNFTAddress as MinterHolderV1UnregisteredNFTAddress,
   RemovedHoldersOfProjects as MinterHolderV1RemovedHoldersOfProjects
 } from "../generated/MinterHolderV1/MinterHolderV1";
+import {
+  AllowedHoldersOfProjects as MinterHolderV2AllowedHoldersOfProjects,
+  RegisteredNFTAddress as MinterHolderV2RegisteredNFTAddress,
+  UnregisteredNFTAddress as MinterHolderV2UnregisteredNFTAddress,
+  RemovedHoldersOfProjects as MinterHolderV2RemovedHoldersOfProjects
+} from "../generated/MinterHolderV2/MinterHolderV2";
+import { DelegationRegistryUpdated as MinterHolderV2DelegationRegistryUpdated } from "../generated/MinterHolderV2/IFilteredMinterHolderV1";
+import { DelegationRegistryUpdated as MinterMerkleV3DelegationRegistryUpdated } from "../generated/MinterMerkleV3/IFilteredMinterMerkleV1";
```

Comment:
Once redeployed, we should double check that this doesn't break. I think since the event is from the interface to begin with it should be fine but since the handlers are for the specific types of contracts I'm worried it might bork.

Reply:
ah, yeah, should probably import from the interface here (I think)...

Reply:
I think this can be resolved, with the agreement that this should be tested on our dev subgraph to ensure nothing breaks. The best way to test this is to ensure we either fully re-sync our dev subgraph without grafting, or ensure if we graft, the starting block is before these two minters were deployed (since they only emit this event during deployment).